src/Service/CloudWatchLogs/tests/Unit/Result/DescribeLogStreamsResponseTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace AsyncAws\CloudWatchLogs\Tests\Unit\Result;
+
+use AsyncAws\CloudWatchLogs\Result\DescribeLogStreamsResponse;
+use AsyncAws\CloudWatchLogs\ValueObject\LogStream;
+use AsyncAws\Core\Response;
+use AsyncAws\Core\Test\Http\SimpleMockedResponse;
+use AsyncAws\Core\Test\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpClient\MockHttpClient;
+
+class DescribeLogStreamsResponseTest extends TestCase
+{
+    public function testDescribeLogStreamsResponse(): void
+    {
+        // see https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_DescribeLogStreams.html
+        $response = new SimpleMockedResponse('{
+           "logStreams": [
+              {
+                 "arn": "test_arn",
+                 "creationTime": 1000,
+                 "firstEventTimestamp": 1001,
+                 "lastEventTimestamp": 1002,
+                 "lastIngestionTime": 1003,
+                 "logStreamName": "test_logStreamName",
+                 "storedBytes": 1004,
+                 "uploadSequenceToken": "test_uploadSequenceToken"
+              }
+           ],
+           "nextToken": "test_nextToken"
+        }');
+
+        $client = new MockHttpClient($response);
+        $result = new DescribeLogStreamsResponse(new Response($client->request('POST', 'http://localhost'), $client, new NullLogger()));
+
+        /** @var LogStream[] $logStreams */
+        $logStreams = iterator_to_array($result->getLogStreams(true));
+
+        self::assertCount(1, $logStreams);
+        self::assertSame('test_arn', $logStreams[0]->getArn());
+        self::assertSame('1000', $logStreams[0]->getCreationTime());
```

Comment:
I wonder why those numbers are converted to strings

this doesn't happen in PutLogEventsResponse

Reply:
This property is defined as `long` in SDK. PHP is not able to (always) use integer to handle theme and have to be converted in string.